scripts/teardown.sh

```diff
@@ -20,41 +20,40 @@ set -x
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 # shellcheck source=scripts/istio.env
 source "$ROOT/scripts/istio.env"
-
-gcloud beta container clusters update "$ISTIO_CLUSTER" \
-    --update-addons=Istio=DISABLED \
-    --project "$ISTIO_PROJECT" \
-    --zone "$ZONE"
+ISTIO_DIR="$ROOT/istio-${ISTIO_VERSION}"
 
 kubectl delete ns vm --ignore-not-found=true
 kubectl delete ns bookinfo --ignore-not-found=true
-kubectl delete svc dns-ilb -n kube-system --ignore-not-found=true
+
+# Uninstall the ILBs installed for mesh expansion
+kubectl delete -f "$ISTIO_DIR/install/kubernetes/mesh-expansion.yaml" --ignore-not-found=true
```

Comment:
We need to wait for this to complete.

Reply:
its already there! i only added a command to delete a couple of firewall rules that occasionally 
 are not deleted when you delete the cluster.